### DISCUSSION TITLE
Download videos that are under a certain length

### DISCUSSION THREAD

**Discussion by samuelbraun04** [Serene-Arc/bulk-downloader-for-reddit#778] [Q&A] [answered]
I have the following yaml file:

```
file_scheme: "{UPVOTES}_{REDDITOR}_{POSTID}_{DATE}"
limit: 50
sort: top
time: day
no_dupes: true
subreddit:
  - Python
```

How can I make sure I only download videos and only if the video is under a certain length (60 seconds for example)

**Comment by Serene-Arc** [comment, marked as answer]
Downloading only videos is possible, but making sure that they are a certain length of have any other characteristic is not. The BDFR doesn't have any of that information on hand.

To download videos, you can use `--skip` to filter out the file types that appear in the subreddit you're looking at so that only the videos download.

**Reply by samuelbraun04** [reply]
Sounds good, thank you.

Just checking: the following code won't necessarily return me 50 videos – it'll just get the top 50 posts on the subreddit and download those that are not of the specified types.
```
skip: [png, jpg, txt, gif, jpeg, bmp, tiff, svg]
file_scheme: "{TITLE}_{POSTID}"
limit: 50
sort: top
time: day
no_dupes: true
subreddit:
  - FunnyAnimals
```

**Reply by Serene-Arc** [reply]
Yes, that's correct.
